contracts/schemes/TokenTrade.sol

```diff
@@ -0,0 +1,159 @@
+pragma solidity ^0.6.10;
```

Comment:
lets make is solidity 0.6.11

Reply:
Seems problematic to push just one file version... maybe push all in a separate PR?

Reply:
ok.